src/components/QuickOpenModal.js

```diff
@@ -130,34 +139,30 @@ export class QuickOpenModal extends Component<Props, State> {
     } = this.props;
 
     let results = functions;
-    results = results.filter(result => result.title !== "anonymous");
```

Comment:
What is the reason for removing this filtering operation from the general case and moving it into the `if (query === "@" || query === "#")` case?

Reply:
@fvsch thanks so much for the question and thorough review.  Honestly this is my mistake.  I was experimenting with this code block to gain deeper understanding, and didn't intend to move the filtering operation for my commit.